tools/hpo/utils/backends.py

```diff
@@ -0,0 +1,132 @@
+"""Sweep Backends."""
```

Comment:
Can `WandbSweep` and `CometSweep` be considered as entrypoint? If not, can these be added to `anomalib/utils/sweep`?

Reply:
Not sure. @djdameln any thoughts?

Reply:
I would suggest either `anomalib/utils/hpo/backends.py` or `anomalib/utils/hpo/runners.py`

Reply:
As long as they are placed into `anomalib/utils/hpo/` or `anomalib/utils/sweep`, I don't mind what the modules are called.

Reply:
@samet-akcay I have moved it to `anomalib/utils/hpo/runners.py`

Reply:
Thanks